core/src/main/java/tech/tablesaw/io/csv/CsvReader.java

```diff
@@ -578,6 +578,9 @@ private ColumnType selectType(List<ColumnType> typeCandidates) {
     private CsvParser csvParser(CsvReadOptions options) {
         CsvParserSettings settings = new CsvParserSettings();
         settings.setFormat(csvFormat(options));
+        if(options.maxNumberOfColumns() != null){
```

Comment:
Can you please add a space after the `if` and before the `{`?

Reply:
fixed